delivery.yaml

```diff
@@ -0,0 +1,26 @@
+version: "2017-09-20"
+pipeline:
+  - id: build
+    type: script
+    commands:
+      - desc: Install Docker
+        cmd: |
+          curl -fLOsS https://delivery.cloud.zalando.com/utils/ensure-docker && sh ensure-docker && rm ensure-docker
+
+      - desc: Build and push Docker image
```

Comment:
do you think it makes sense to run tests from cdp? I see that travis file exists in the repo, which runs the tests.

Reply:
True, makes sense